src/main/resources/run-class.sh

```diff
@@ -100,7 +100,7 @@ fi
 
 #Add a cache-key config, user tests for existence
 ADD_CACHE_KEY=""
-if [ -e conf/cacheKey.Sha1 ]; then
+if [ -e ../conf/cacheKey.Sha1 ]; then
```

Comment:
This isn't correct. See line 56 - this changes to the directory above `bin`, which contains `conf`. Note that all of the other configuration paths use `conf` as the base . . .